cmd/awsToken.go

```diff
@@ -118,7 +118,7 @@ func lookupAccountAliases(arns []Arn, assertion string, cache *AccountAliasCache
 			} else {
 				// Print the error, cast err to awserr.Error to get the Code and
 				// Message from an error.
-				journal("Unable to resolve account alias", err.Error())
```

Comment:
why don't you keep the error?

Reply:
It was my first try to use go and my IDE said that there are to much Parameters used in method journal, so I deleted it ;-)